tests/test_mgxs_library_hdf5/results_true.dat

```diff
@@ -105,18 +105,18 @@ domain=10000 type=prompt-nu-fission matrix
 [[3.14909051e-03 0.00000000e+00]
  [2.86750878e-02 0.00000000e+00]]
 domain=10000 type=delayed-nu-fission
-[[2.29808266e-05 1.06974147e-04]
```

Comment:
While the only *test* results that changed use delayed groups, other filters will have similar effects if misaligned so there could be other *user* results affected. In my work, the misalignment only had a subtle effect on large problems and was only really obvious when I looked at a IHM one-group case.